bin/resources/GameIndex.yaml

```
@@ -13572,12 +13614,12 @@ SLES-52363:
   region: "PAL-G"
 SLES-52366:
   name: "America's 10 Most Wanted"
-  region: "PAL-M5"
+  region: "PAL-UK"
```

```suggestion
  region: "PAL-M5"
```
it was already correct

---

http://redump.org/disc/4487/

---

No it isn't. GameIndex asks for the game region, not the languages. So this one is PAL-UK, no matter how many languages it contains... No ?

---

@Mrlinkwii @refractionpcsx2 Maybe we should write down some guidelines about titles and regions and insert them in the wiki so we don't loose any extra time about subjects already settled?

---

No, it's based on the region AND languages.  if it has 5 languages and it's europe, it's PAL-M5, for Multi language (5). same for 3 language it'd be PAL-M3

---

Sorry to be THAT insistent...
10 Most Wanted (known in the US as "Fugitive Hunter: War on Terror") has 3 versions in PAL area :
* SLES-52367 [America's 10 Most Wanted](http://redump.org/disc/4294/) which is the PAL M5
* SLES-52856 [Most Wanted](http://redump.org/disc/51987/) which was the french version sold in France (and that had M5 too)
* SLES-52366 [America's 10 Most Wanted](http://redump.org/disc/4487/) which was the UK version (it had the BBFC 15 logo on [its cover](http://www.computinghistory.org.uk/userdata/images/large/22/84/product-82284.jpg) and had the M5 too, like the french version)

---

no, it has 5 languages, so it's M5, I don't care if there's already one M5 version, this is another one.

---

That's why I talked about some guidelines. I'm fine with setting this one to PAL-M5 but we should write that down somewhere so next contributors know that languages are more important than region.

Could it be possible to have both region and languages in GameIndex.yaml ?
For example, if there's no languages in the "languages" line, the region is the default language.

Example :
`SLES-52366:
  name: "America's 10 Most Wanted"
  region: "PAL-M5"
  languages: E,F,G,I,S`

or

`SLES-52366:
  name: "Akudaikan 3"
  region: "NTSC-J"`
-> no line for language = J

---

That is for the future to make it region for real and languages seperately. But GameDB is always a pain.

---

I'd rather wait for this feature to be real so, because it takes a very long time to check every single title... And I don't want to double that job, once it will be implemented ^_^;